astropy/stats/bayesian_blocks.py

```diff
@@ -472,20 +487,18 @@ class PointMeasures(FitnessFunc):
     Parameters
     ----------
     p0 : float (optional)
-        False alarm probability, used to compute the prior on N
-        (see eq. 21 of Scargle 2012).  Default prior is for p0 = 0.
+        False alarm probability, used to compute the prior on N_blocks
```

Comment:
As per above, you could do ``:math:`N_{\rm blocks}` `` (although in this case it's not really that necessary as it's easy enough to read as-is).